save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/views/demo/CpgView.kt

```diff
@@ -95,14 +95,24 @@ val cpgView: VFC = VFC {
                                 }
                             }
                             div {
-                                val alertStyle = if (queryId.isNotBlank()) "alert-primary" else ""
+                                val alertStyle = if (cpgResult.applicationName.isNotBlank()) "alert-primary" else ""
                                 className = ClassName("alert $alertStyle text-sm mt-3 pb-2 pt-2 mb-0")
-                                +queryId
+                                +cpgResult.applicationName
                             }
                         }
                     }
+                    this.showLogs = { setIsLogs(it) }
                 }
             }
         }
     }
+    if (isLogs && cpgResult.logs.isNotEmpty()) {
```

Comment:
maybe logs empty logs when `isLogs == true`, but `cpgResult.logs.isEmpty()`?

Reply:
No logs provided and `alert-secondary` in this case instead of `alert-primary`

Reply:
May be. Then the user will see an empty block for logs.

Reply:
@Cheshiriks I meant to display it
> No logs provided